### ISSUE TITLE
Migrate: vega.xyz

### ISSUE THREAD

**Issue by edd** [vegaprotocol/vega.xyz#674]
Blocked by: https://github.com/vegaprotocol/vega.xyz/issues/673

- [x] Set up Vercel site
- [x] Set up redirects
- [x] Repoint E2E tests in github action
- [x] Wait for #673 
- [x] Lock netlify site

Next: 
Make terraform update (separate ticket)

Later: 
- [x] Remove site from Netlify
